build_product

```diff
@@ -93,6 +94,8 @@ print_help()
 	printf '\t%s\n' "--derivatives, --no-derivatives: Also build derivatives of products if applicable (off by default)"
 	printf '\t%s\n' "--ansible-playbooks, --no-ansible-playbooks: Build Ansible Playbooks for every profile (on by default)"
 	printf '\t%s\n' "--bash-scripts, --no-bash-scripts: Build Bash remediation scripts for every profile (on by default)"
+	printf '\t%s\n' "-t, --thin, --no-thin: Build thin data streams for each rule. Do not build any of the guides, tables, etc (off by default)"
```

Comment:
This takes a horrible amount of time. Do you have any idea why it could take so long?

Reply:
Because it generates about 1800 thin ds.

Reply:
Generating the XCCDF benchmark on a profile and linking it with other components causes the biggest slowdown.  I'm working on speeding it up right now. Should I incorporate the changes into this PR or create another one?

Reply:
I propose doing that in a separate PR. This PR is already large enough.